docs/source/references/functional.rst

```diff
@@ -270,6 +270,12 @@ peak_signal_noise_ratio [func]
 .. autofunction:: torchmetrics.functional.peak_signal_noise_ratio
     :noindex:
 
+universal_image_quality_index [func]
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
The length of `~` here should be equal to the length of the preceeding string 
`universal_image_quality_index [func]`

Reply:
Oh, my bad. Thanks for pointing this out, I have fixed this now :)